app/controllers/attended_events_controller.rb

```diff
@@ -0,0 +1,34 @@
+class AttendedEventsController < ApplicationController
+  before_action :logged_in_user, only: [:create]
+  before_action :check_subscription_status, only: [:create]
+
+  
+  def create
+    @user = current_user().id
+    @attend = AttendedEventsTbl.new(attendee_id: @user, attended_event_id: params[:event])
+    @attend.save
+    flash[:success] = "Succesfully subscribed to the event."
+    redirect_to "/users/#{@user}"
```

Comment:
the common practice is to use an users_path method with user passed as an argument 
``` redirect_to users_path current_user ``` in this case

Reply:
corrected...thanks